.htaccess

```diff
@@ -29,8 +29,8 @@ ErrorDocument 500 /assets/error-500.html
 <IfModule mod_rewrite.c>
 	RewriteEngine On
 
-	RewriteCond %{HTTP_HOST} ^docs?.silverstripe.com$ [NC]
-    RewriteRule ^(.*)$ https://docs.silverstripe.org/$1 [L,R=301]
+	RewriteCond %{HTTP_HOST} ^docs?.silverstripe.org$ [NC]
```

Comment:
the condition is if request comes in on our .com domain eg docs.silverstripe.com it redirects to the correct docs.silverstripe.org domain. What you have here will create an infinite loop when you get it on the actual production server.